### ISSUE TITLE
Incorrect popover position on scaled body element

### ISSUE THREAD

**Issue by egnawake** [ng-bootstrap/ng-bootstrap#3724]
### Bug description:
When applying a scale transform to the body element, popovers are positioned incorrectly. This is very noticeable on the "Popover on right" example.

### Link to minimally-working StackBlitz that reproduces the issue:
https://stackblitz.com/edit/angular-hbbcrb

#### My environment:
* Angular: 7.1.0
* ng-bootstrap: 4.1.1
* Bootstrap: 4.1.3

#### The StackBlitz example:
* Angular: 9.0.7
* ng-bootstrap: 6.1.0

**Comment by benouat** [comment]
@fbasso could you please take a look ?

**Comment by fbasso** [comment]
I don't know if we can do something with the scale. It seems it's calculating position with the scale is not easy. This is what is displayed with the scale on body, with the official Bootstrap and popper:

![image](https://user-images.githubusercontent.com/1152930/82193841-411cba80-98f6-11ea-921c-1fb75abaa999.png)

**Comment by fbasso** [comment]
It's really hard to understand how the borwser calculate the translation here. If I set it to (0, 0), Firefox shows a translation anyway:

![image](https://user-images.githubusercontent.com/1152930/82301241-3f1b3000-99b8-11ea-999d-ea24e3e68089.png)

(here, it's visually at the right place, as it's position at (0, 0) of the container. But if I set it to the bottom right of the container, the (basically the width and height of the container), it's badly positioned:

![image](https://user-images.githubusercontent.com/1152930/82301495-a46f2100-99b8-11ea-84a2-cb6e994a53d8.png)

It seems to be a browser issue with the calculation, or something hard to understand about the calculation.
